doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -766,58 +726,81 @@ the conversion in hypotheses :n:`{+ @ident}`.
          pattern (0 = 0).
          fold not.
 
-   .. tacv:: fold {+ @term}
+.. tacn:: pattern {+, @pattern_occs } {? @occurrences }
 
-      Equivalent to :n:`fold @term ; ... ; fold @term`.
+   .. todo PR: how do multiple pattern_occs work with @occurrences??
 
-.. tacn:: pattern @term
-   :name: pattern
+   Performs :term:`eta-expansion`.  If :n:`@occurrences` is given,
+   the specified occurrences will be replaced in the selected hypotheses and/or goal.
+   If :n:`@occurrences` is not given, every occurrence of the resulting term in the goal is
```

Comment:
Doesn't `pattern_occs` permit `at` clauses to restrict the occurrences?

Reply:
Also, what is "the resulting term"?

Reply:
I think no longer applies.